staticaddr/deposit/fsm.go

```diff
@@ -33,6 +33,14 @@ var (
 
 	Withdrawn = fsm.StateType("Withdrawn")
 
+	LoopingIn = fsm.StateType("LoopingIn")
+
+	LoopedIn = fsm.StateType("LoopedIn")
+
+	SweepHtlcTimout = fsm.StateType("SweepHtlcTimout")
```

Comment:
Typo: Timout -> Timeout

Also other instances in this PR (OnSweepingHtlcTimout).

Reply:
I think I got all.